lib/exercism/user.rb

```diff
@@ -21,6 +21,12 @@ class User < ActiveRecord::Base
 
   before_save do
     self.key ||= Exercism.uuid
+
+    # migrate acls when track_mentor changes
+    if track_mentor_changed?
```

Comment:
I'd like to avoid using a callback for this. I'd rather just explicitly update the acls where we are making the change.